### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by Nicxzmiller**
Created Navigation Component

### CODE REVIEW

File: .idea/fun-react-lib.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ashispavan) [inline .idea/fun-react-lib.iml:1]: Is this an IDE workspace file?

Comment (NehemiahK) [reply .idea/fun-react-lib.iml:1]: Good catch, this should not be in there. Can you change this?

**Comment by NehemiahK** [comment]
Can you resolve these conflicts @Nicxzmiller

**Comment by Nicxzmiller** [comment]
Yes, will do. Thanks for bringing it to my attention

**Comment by Caleb335** [comment]
Fixed conflicting files

**Comment by Nicxzmiller** [comment]
Thanks Caleb. Saw it.

File: .idea/fun-react-lib.iml (hunk 1 above)
Comment (KrishnaMoorthy12) [reply .idea/fun-react-lib.iml:1]: @Nicxzmiller, you're currently working on removing this?

**Pull request merged**